lib/index.js

```diff
@@ -1,16 +1,19 @@
-const path = require('path')
+const Debug = require('debug')
+const { sep, resolve, basename } = require('path')
 const rollup = require('rollup')
 
+const debug = Debug('metalsmith-rollup')
```

Comment:
Let's not add debug support just like this, or if we think it's needed, let's add debugging to all key areas.